tidb-cloud/tidb-cloud-intro.md

```diff
@@ -6,42 +6,60 @@ category: intro
 
 # TiDB Cloud Introduction
 
-[TiDB Cloud](https://pingcap.com/products/tidbcloud) is a fully-managed Database-as-a-Service (DBaaS) that brings everything great about TiDB to your cloud, and lets you focus on your applications, not the complexities of your database.
+[TiDB Cloud](https://en.pingcap.com/tidb-cloud/) is a fully-managed Database-as-a-Service (DBaaS) that brings [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open-source Hybrid Transactional and Analytical Processing (HTAP) database, to your cloud. TiDB Cloud offers an easy way to deploy and manage databases to let you focus on your applications, not the complexities of the databases. You can create TiDB Cloud clusters to quickly build mission-critical applications on Google Cloud Platform (GCP) and Amazon Web Services (AWS).
+
+![TiDB Cloud Overview](/media/tidb-cloud/tidb-cloud-overview.png)
+
+## Why TiDB Cloud
+
+TiDB Cloud allows you with little or no training to handle complex tasks such as infrastructure management and cluster deployment easily.
+
+- Developers and database administrators (DBAs) can rapidly analyze a large volume of data across multiple datasets.
```

Comment:
Developers and database administrators (DBAs) can handle large amount of online traffic effortlessly as well as rapidly analyze a large volume of data across multiple datasets.